### PR TITLE
Temporary fix for OpenSim config issue for downstream projects

### PR DESCRIPTION
**Pull request by klshrinidhi**
This allows the downstream projects to link against OpenSim including OpenSim's dependencies (BTK & Simbody).
I successfully built a test project on Linux and Windows in Debug, Release and RelWithDebInfo.

### CODE REVIEW

**Comment by klshrinidhi** [comment]
PR #1000 !!!!!!!

File: cmake/OpenSimConfig.cmake.in [hunk 1]
```diff
@@ -35,6 +35,8 @@ set_and_check(@CMAKE_PROJECT_NAME@_ROOT_DIR "@PACKAGE_CMAKE_INSTALL_PREFIX@")
 # -------
 set_and_check(@CMAKE_PROJECT_NAME@_LIB_DIR "@PACKAGE_CMAKE_INSTALL_LIBDIR@")
 
+list(APPEND @CMAKE_PROJECT_NAME@_LIB_DIR "@BTK_LIBRARY_DIRS@")
+
```
Comment (chrisdembia) [inline cmake/OpenSimConfig.cmake.in]: Based on https://github.com/opensim-org/opensim-gui/pull/40 and https://github.com/opensim-org/opensim-gui/pull/40/files#diff-b8f1383df0dfd383eb189adf34c5d2abR39, it seems like `OpenSim_LIB_DIR` should contain only one path. Could we create a new variable that holds a list of paths? Like `OpenSim_DEPS_LIB_DIRS` for example? or `OpenSim_AND_DEPS_LIB_DIRS`?

File: cmake/OpenSimConfig.cmake.in [hunk 2]
```diff
@@ -75,3 +77,7 @@ set(@CMAKE_PROJECT_NAME@_JAVA_FILES_DIR
 set(@CMAKE_PROJECT_NAME@_JAR_FILE 
     @PACKAGE_CMAKE_INSTALL_PREFIX@/@OPENSIM_INSTALL_JAVAJARDIR@/@SWIG_JAVA_JAR_NAME@)
 
+find_package(Simbody ${SIMBODY_VERSION_TO_USE}
+    PATHS "@SIMBODY_HOME@" NO_MODULE NO_DEFAULT_PATH)
```
Comment (chrisdembia) [inline cmake/OpenSimConfig.cmake.in]: This is correct if `OPENSIM_COPY_SIMBODY` is OFF, but if `OPENSIM_COPY_SIMBODY` is ON, then we should be using the Simbody that is packaged with OpenSim.

**Comment by chrisdembia** [comment]
Given that this is a temporary fix, I'm fine merging this as is once the CI tests pass. Thanks for testing on all those platforms.

**Comment by chrisdembia** [comment]
Also, I think we can ignore the `appveyor/branch` failure...it seems like a random failure and the `appveyor/pr` build is what actually matters. So, just waiting on the `travis-ci/pr` build.

**Comment by chrisdembia** [comment]
Thanks for doing this. We should revisit soon after the workshop.

**Pull request merged**